src/bundle/Resources/public/js/scripts/admin.limitation.pick.js

```diff
@@ -1,19 +1,164 @@
 (function(global, doc, eZ, React, ReactDOM, Translator) {
-    const udwContainer = doc.getElementById('react-udw');
+    const SELECTOR_LOCATION_LIMITATION_BTN = '.ez-pick-location-limitation-button';
+    const SELECTOR_EZ_TAG = '.ez-tag';
+    const IDS_SEPARATOR = ',';
     const token = doc.querySelector('meta[name="CSRF-Token"]').content;
     const siteaccess = doc.querySelector('meta[name="SiteAccess"]').content;
-    const closeUDW = () => ReactDOM.unmountComponentAtNode(udwContainer);
-    const selectLocationsConfirm = (target, data) => {
-        const selectedItems = data.reduce((total, item) => total + `<li>${item.ContentInfo.Content.Name}</li>`, '');
+    const udwContainer = doc.getElementById('react-udw');
+    const limitationBtns = doc.querySelectorAll(SELECTOR_LOCATION_LIMITATION_BTN);
+    const findLocationsByIdList = (pathArraysWithoutRoot, callback) => {
+        const bulkOperations = getBulkOperations(pathArraysWithoutRoot);
+        const request = new Request('/api/ezp/v2/bulk', {
+            method: 'POST',
+            headers: {
+                Accept: 'application/vnd.ez.api.BulkOperationResponse+json',
+                'Content-Type': 'application/vnd.ez.api.BulkOperation+json',
+                'X-Siteaccess': siteaccess,
+                'X-CSRF-Token': token,
+            },
+            body: JSON.stringify({
+                bulkOperations: {
```

Comment:
Does `bulkOperations` object contain only `operations` property or it can have other properties?

Reply:
It has only `operations` property, as described in https://github.com/ezsystems/ezplatform-admin-ui/pull/691 
It seems that this have too stay as it is because of something. I don't know exact details - backend.

Reply:
Thanks for explaining it. Backend and everything makes sense now :P